src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.PowerToys/Components/Utility.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Windows.Input;
+using Common.UI;
+using interop;
+using Wox.Infrastructure;
+using Wox.Plugin;
+
+namespace Microsoft.PowerToys.Run.Plugin.PowerToys.Components
+{
+    public class Utility
+    {
+        public UtilityKey Key { get; }
+
+        public string Name { get; }
+
+        public bool Enabled { get; set; }
+
+        public Utility(UtilityKey key, string name, bool enabled)
+        {
+            Key = key;
+            Name = name;
+            Enabled = enabled;
+        }
+
+        public Result CreateResult(MatchResult matchResult)
+        {
+            return new Result
+            {
+                Title = Name,
+                SubTitle = "PowerToys Utility",
+                IcoPath = UtilityHelper.GetIcoPath(Key),
+                Action = UtilityHelper.GetAction(Key),
+                ContextData = this,
+                Score = matchResult.Score,
+                TitleHighlightData = matchResult.MatchData,
+            };
+        }
+
+        public List<ContextMenuResult> CreateContextMenuResults()
+        {
+            var results = new List<ContextMenuResult>();
+
+            if (Key == UtilityKey.Hosts)
+            {
+                results.Add(new ContextMenuResult
+                {
+                    Title = "Run as administrator (Ctrl+Shift+Enter)",
+                    Glyph = "\xE7EF",
+                    FontFamily = "Segoe MDL2 Assets",
```

Comment:
Please don't hard-code this. Rather use {StaticResource}

Reply:
All strings will be localized.

Reply:
Uhm, what do you mean? This is not a visible text. The icon font MDL2 should not be hardcoded, and on Windows 11 it's inappropriate.

Reply:
I meant title.
This is how MDL2 icons are set in every PT Run plug-ins. What's wrong with Windows 11?

Reply:
Windows 11 uses a different icon font: Segoe Fluent. We don't want to make icons in PowerToys inconsistent on Windows 11 by using Windows 10 icons, right?

Reply:
AFAIK Windows 11 still render MDL2 assets in fact in PT Run icons are working.
This solution is used across multiple plugins.
Can we limit the review to the new feature to keep it clean?

![image](https://user-images.githubusercontent.com/25966642/226750507-b4504635-481f-4de3-bf13-7b11c01f9ecb.png)

Reply:
The resource `FontFamily="{StaticResource SymbolThemeFontFamily}"` is available in WinUI3 and PowerToys Run is on WPF.
AFAIK no issues have been reported with context action icons in Windows 11.
Still no related to the PR.